src/foam/core/Validation.js

```diff
@@ -102,7 +102,7 @@ foam.CLASS({
         }
         return !required ? null : [[name],
           function() {
-            return !this.hasOwnProperty(name) && (`Please enter a ${label}`);
+            return !this.hasOwnProperty(name) && (`Please enter a ${label.toLowerCase()}`);
```

Comment:
`Please provide a value for ${label.toLowerCase()}` avoids the a vs an problem.

Reply:
Even simpler is "This field cannot be blank" or "This field is required". What do you think about that instead?

Reply:
required fields will have a red boarder and red star underneath with this text. 

> * required
 
I think is sufficient. No issues with locale, property names. ... 
The placeholder text can handle provide a template/default/example.

Reply:
required fields will have a red boarder and red star underneath with this text. 

> * required
 
I think is sufficient. No issues with locale, property names. ... 
The placeholder text can handle provide a template/default/example.